sdk/servicebus/Azure.ResourceManager.ServiceBus/api/Azure.ResourceManager.ServiceBus.netstandard2.0.cs

```diff
@@ -201,11 +201,11 @@ public partial class ServiceBusNamespaceData : Azure.ResourceManager.Models.Trac
         public Azure.ResourceManager.Models.ManagedServiceIdentity Identity { get { throw null; } set { } }
         public bool? IsZoneRedundant { get { throw null; } set { } }
         public string MetricId { get { throw null; } }
-        public Azure.ResourceManager.ServiceBus.Models.ServiceBusMinimumTlsVersion? MinimumTlsVersion { get { throw null; } set { } }
+        public Azure.ResourceManager.ServiceBus.Models.TlsVersion? MinimumTlsVersion { get { throw null; } set { } }
         public int? PremiumMessagingPartitions { get { throw null; } set { } }
         public System.Collections.Generic.IList<Azure.ResourceManager.ServiceBus.ServiceBusPrivateEndpointConnectionData> PrivateEndpointConnections { get { throw null; } }
         public string ProvisioningState { get { throw null; } }
-        public Azure.ResourceManager.ServiceBus.Models.ServiceBusPublicNetworkAccess? PublicNetworkAccess { get { throw null; } set { } }
+        public Azure.ResourceManager.ServiceBus.Models.PublicNetworkAccess? PublicNetworkAccess { get { throw null; } set { } }
```

Comment:
These are unexpected changes. Please rebase with the main branch and regen the SDK.